x/pairing/keeper/pairing.go

```diff
@@ -283,7 +283,7 @@ func (k Keeper) ValidatePairingForClient(ctx sdk.Context, chainID string, provid
 	}
 
 	for _, possibleAddr := range validAddresses {
-		providerAccAddr, err := sdk.AccAddressFromBech32(possibleAddr.Address)
+		providerAccAddr, err := sdk.AccAddressFromBech32(possibleAddr.Operator)
```

Comment:
what about the vault address?

Reply:
pairing is done only with the operator addresses